File: cpp/Processor.cpp
#include <iostream>

using namespace std;

class Processor
{
private:
    string name;
    int price;

public:
    Processor() {}
    Processor(string name, int price)
    {
        this->name = name;
        this->price = price;
    }

    string getName()
    {
        return this->name;
    }
    void setName(string name)
    {
        this->name = name;
    }

    int getPrice()
    {
        return this->price;
    }
    void setPrice(int price)
    {
        this->price = price;
    }

    void printProcessor()
    {
        cout << "Processor Name: " << this->name << endl;
        cout << "Processor Price: " << this->price << endl;
    }
    ~Processor() {}
};

File: python/Processor.py
class Processor:
    def __init__(self, name="", price = 0):
        self.name = name
        self.price = price

    def getName(self):
        return self.name
    
    def setName(self, name):
        self.name = name

    def getPrice(self):
        return self.price

    def setPrice(self, price):
        self.price = price

    def printProcessor(self):
        print("Processor Name: {}".format(self.name))
        print("Processor Price: {}".format(self.price))